### PR TITLE
Fix nxplayer/nxrecoder/nxlooper warning

### PR DESCRIPTION
**Pull request by qiaohj213**
## Summary
nxlooper, nxplayer:   fix warning

## Impact
nxlooper, nxplayer

## Testing
vela

### CODE REVIEW

File: system/nxplayer/nxplayer.c [hunk 1]
```diff
@@ -314,7 +314,7 @@ static int nxplayer_opendevice(FAR struct nxplayer_s *pplayer, int format,
 
 #ifdef CONFIG_NXPLAYER_INCLUDE_DEVICE_SEARCH
     {
-      struct audio_caps_s caps;
+      struct audio_caps_s cap;
```
Comment (pkarashchenko) [inline system/nxplayer/nxplayer.c:317]: what is the warning reposted here?

Comment (qiaohj213) [reply system/nxplayer/nxplayer.c:317]: checkpatch.sh failed on Line:398,  the reason is "found Long Line".  So change caps to cap.

File: system/nxplayer/nxplayer.c [hunk 2]
```diff
@@ -385,25 +385,25 @@ static int nxplayer_opendevice(FAR struct nxplayer_s *pplayer, int format,
...
                            */
 
-                          caps.ac_subtype = format;
-                          caps.ac_format.b[0] = 0;
+                          cap.ac_subtype = format;
+                          cap.ac_format.b[0] = 0;
 
                           while (ioctl(pplayer->dev_fd, AUDIOIOC_GETCAPS,
-                              (unsigned long) &caps) == caps.ac_len)
+                              (unsigned long) &cap) == cap.ac_len)
```
Comment (pkarashchenko) [inline system/nxplayer/nxplayer.c:392]: optional
```suggestion
                                 (unsigned long)&cap) == cap.ac_len)
```

File: system/nxplayer/nxplayer.c (hunk 1 above)
Comment (pkarashchenko) [reply system/nxplayer/nxplayer.c:317]: Ok. Tricky way to handle long lines :), but still it works. Usually people do
```
for (x = 0;
     x < sizeof(caps.ac_controls);
     x++)
```

**Review by pkarashchenko — APPROVED** [review]

**Pull request merged**